Automattic/jetpack
# Copy Post: Add a Jetpack logo to row action link

## Proposed changes:

* Adds Jetpack's logo to the "Copy" row action link. This makes it more obvious that the Copy post functionality is being provided by Jetpack instead of Core.

Before|After
-|-
![image](https://github.com/Automattic/jetpack/assets/4044428/300f72e9-ed6f-47d2-8f9b-78e5872ac19e)|![image](https://github.com/Automattic/jetpack/assets/4044428/0ffbb3ee-8d91-4033-8b7c-a0fa95199d7c)

### Other information:


## Jetpack product discussion


Prior verbal discussions.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Apply this change to your site with Jetpack.
* Enable the Copy Post module, if necessary (`/wp-admin/admin.php?page=jetpack_modules&s=copy`).
* Navigate to Posts (`/wp-admin/edit.php`) or Pages (`/wp-admin/edit.php?post_type=page`) in the sidebar.
* Hover over one of your Posts or Pages.
* Ensure you see a Jetpack logo within the "Copy" row action link.
